hubot-scripts/hubot-readme.md

```diff
@@ -0,0 +1,455 @@
+
+I created a team to write this blog post. You might already have hubot installed at work, etc. For my little slack team, I followed the “Integrating Hubot with Slack” instructions from https://www.sitepoint.com/spice-up-your-slack-channel-with-hubot/ to get Hubot installed. 
+
+Follow the section “Where to Find Pre-built Scripts” to make sure you can run other scripts, like pug me, which is the real equivalent of Hello World in the Hubot world.
+
+Next, you want to read the section “Building a Script to Reduce Employee Efficiency” which we already did in a way because we have our bb8.coffeescript already installed in the scripts directory. Planning!
+
+Now jump to the section “To test this, start up Hubot using:” Here’s where I got lost. I didn’t realize that once you run this command,
+
+`HUBOT_SLACK_TOKEN=YOUR_SLACK_API_TOKEN_HERE ./bin/hubot --adapter slack`
+
+Hubot appears online in your slack team. I tried to run this command several times, so I got “port in use” node errors, until I realized, “oh, the robot is working as expected. The human is confused…”
+
+And now putting it all together!
+
+In your local hubot test-bot directory, run
+
+`HUBOT_SLACK_TOKEN=YOUR_SLACK_API_TOKEN_HERE ./bin/hubot — adapter slack`
+
+**Note:** that you won’t get a command prompt when you hit this time in your terminal because hubot is running in slack.
+
+You should see hubot online in slack
+
+![yay! hubot’s here in slack!](https://cloud.githubusercontent.com/assets/11529908/16711599/991f23a6-4618-11e6-9408-4bd42d179879.png)
+
+  2.Run the Express server (part 4)
+  3.Run a few cURL color requests to keep the connection to bb-8 active (part 4)
+  4.And now start the drumroll… Go to hubot in your direct messages and type
+
+![hey hubot ask BB-8 about those dance moves!](https://cloud.githubusercontent.com/assets/11529908/16711601/c209dfe0-4618-11e6-82c4-246d08f322f2.png)
+
+Here’s the video I posted earlier this morning.
+
+[![Hubot ask BB-8 if he's got any dance moves](https://cloud.githubusercontent.com/assets/11529908/16711633/5c597c76-461a-11e6-843c-29c0d12eca2d.png)](https://twitter.com/saraford/status/750768569344860160)
+
+Enjoy!
+
+## P.S.
+
+Sometimes BB-8 doesn’t want to go back into his box. If he’s still glowing orange (or he recently stopped glowing), he’s still going to try to keep his head in the same position. You can try to connect to him via the Sphero BB-8 iPhone app and then kill the app to try to convince him it’s bedtime. I’ve tried running scripts to disconnect BB-8, but he doesn’t always buy it. Eventually he’ll power off, but you might have to manually carry him out of the coffee shop.
+
+And remember…
+
+I for one welcome our new BB-8 robot overlords!
+
+![yay bb-8](https://cloud.githubusercontent.com/assets/11529908/16711606/24b2640a-4619-11e6-855c-a87e47ea95f6.png)
+
```

Comment:
The README is very lengthy and seems similar if not the same as the Blog post you posted on Medium. It would be great if we can somehow condense it into easy to digest steps. It's also referencing some coffee script files which you have now removed so it might confuse the person setting this up if its referencing files that don't exist anymore.